authors/apps/article/views.py

```diff
@@ -56,52 +54,10 @@ def get(self, request):
         Retrieve all articles
         """
         article = Article.objects.all()
-        serializer = GetArticleSerializer(article, many=True)
+        serializer = ArticleSerializer(article, many=True, context={'request': self.request})
         return Response(serializer.data)
 
 
-    def destroy(self, request, slug):
```

Comment:
Also, @Kibetchirchir why are we removing these methods? Are they catered for in another class?

Reply:
The destroy and put method are not being called. I refactored the code from generics to API view and forgot to remove the unused methods

Reply:
okay, Thanks